### PR TITLE
Add initial project structure

### PR DESCRIPTION
**Pull request by ok2ju**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# super-enigma-service
\ No newline at end of file
+# super-enigma-service
+
+## Prerequisites
```
Comment (ok2ju) [inline README.md]: delete empty "Prerequisites" or fill it

File: index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  adult: Boolean,
+  backdrop_path: String,
+  video: Boolean,
+});
+
+const Movie = mongoose.model('Movie', movieSchema);
+
+mongoose.connect(process.env.DB_URI, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
```
Comment (ok2ju) [inline index.js:35]: do you "useUnifiedTopology"?

File: index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  console.error(error);
+});
+
+mongoose.connection.once('open', () => {
+  console.log('Connected to MongoDB');
+
+  router
+    .get('/movies', async (ctx, next) => {
+      const { page, perPage } = ctx.query;
+      console.log('params', page, perPage);
```
Comment (ok2ju) [inline index.js]: delete console.log

File: index.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      });
+      ctx.status = 200;
+      ctx.body = movies;
+
+      next();
+    });
+
+  app
+    .use(router.routes())
+    .use(router.allowedMethods());
```
Comment (ok2ju) [inline index.js]: do you need "allowedMethods"?

**Review by menkuichan — APPROVED** [review]

**Pull request merged**